src/fu-util-bios-attr.c

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (C) 2022 Mario Limonciello <mario.limonciello@amd.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#define G_LOG_DOMAIN "FuMain"
+
+#include "config.h"
+
+#include <glib/gi18n.h>
+
+#include "fu-util-bios-attr.h"
+#include "fu-util-common.h"
+
+static gboolean
+fu_util_bios_attr_dup_fields(const gchar *name, const gchar *desc)
+{
+	return g_strcmp0(name, desc) == 0;
+}
+
+static void
+fu_util_bios_attr_update_description(FwupdBiosAttr *attr)
+{
+	const gchar *name = fwupd_bios_attr_get_name(attr);
+	const gchar *old = fwupd_bios_attr_get_description(attr);
+	const gchar *new = NULL;
+
+	if (g_strcmp0(name, FWUPD_BIOS_ATTR_PENDING_REBOOT) == 0) {
+		/* TRANSLATORS: Settings refers to BIOS settings in this context */
+		new = _("Settings will apply after system reboots");
+	}
+	/* For providing a better description on a number of Lenovo systems */
+	if (fu_util_bios_attr_dup_fields(name, old)) {
```

Comment:
else if?

Reply:
actually don't need else if clause here - new can be set multiple ways